src/encord_active/frontend/src/components/explorer/Explorer.tsx

```diff
@@ -405,276 +428,296 @@ export function Explorer({
         }
         editUrl={editUrl}
       />
-      <ExplorerDistribution
-        projectHash={projectHash}
-        predictionHash={predictionHash}
-        filters={filters}
-        addFilter={(domain, metric, min, max) => {
-          const setFilters =
-            domain === "data" ? setDataFilters : setAnnotationFilters;
-          setFilters((filters) => {
-            if (metric in filters.metricFilters) {
-              return {
-                ...filters,
-                metricFilters: {
-                  ...filters.metricFilters,
-                  [metric]: [min, max],
-                },
-              };
-            } else {
-              return {
-                ...filters,
-                metricFilters: {
-                  [metric]: [min, max],
-                  ...filters.metricFilters,
-                },
-                ordering: [...filters.ordering, metric],
-              };
-            }
-          });
-        }}
-      />
-      <Space wrap>
-        {predictionHash !== undefined && (
-          <PredictionFilters
-            disabled={!!similarityItem}
-            iou={iou}
-            setIou={setIou}
-            predictionOutcome={predictionOutcome}
-            isClassificationOnly={false}
-            setPredictionOutcome={setPredictionOutcome}
-          />
-        )}
-        <Space.Compact size="large">
-          <Select
-            value={`${selectedMetric.domain}-${selectedMetric.metric_key}`}
-            onChange={(strKey: string) => {
-              const [domain, metric_key] = strKey.split("-");
-              setSelectedMetric({
-                domain: domain as "data" | "annotation",
-                metric_key,
-              });
+      <Row className="h-full">
+        <Col span={18} className="h-full">
+          <Tabs
+            activeKey={activeView}
+            onChange={setActiveView as (val: string) => void}
+            style={{
+              height: "100%",
+            }}
+            className="h-full"
+            tabBarStyle={{
+              margin: 0,
+              padding: "0px 10px",
             }}
-            className="w-80"
-            options={[
+            tabBarExtraContent={{
+              left: (
+                <ExplorerPremiumSearch
+                  premiumSearchState={{
+                    ...premiumSearchState,
+                    setSearch: (args) => {
+                      setSimilarityItem(undefined);
+                      premiumSearchState.setSearch(args);
+                    },
+                  }}
+                />
+              ),
+              right: (
+                <Segmented
+                  selected
+                  value={metricDomain}
+                  options={["Data", "Label"]}
+                  onChange={
+                    handleSelectedDomainChange as (val: SegmentedValue) => void
```

Comment:
Let's avoid casting types, if typescript says the type is wrong it probably is 😄